src/main/java/com/example/onlinebookstore/OnlineBookStoreApplication.java

```diff
@@ -1,13 +1,30 @@
 package com.example.onlinebookstore;
 
+import com.example.onlinebookstore.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class OnlineBookStoreApplication {
+    @Autowired
+    private BookService bookService;
```

Comment:
You should use a constructor injection. Simply add the @RequiredArgsConstructor annotation above the class and make the field final